tests/test_header_parser.py

```diff
@@ -0,0 +1,74 @@
+from unittest import TestCase, main
+
+from grblogtools import get_dataframe
+from grblogtools.helpers import ParserFlag
+from grblogtools.header_parser import HeaderLogParser
+
+
+class TestHeaderLog(TestCase):
+    def setUp(self):
+        self._path_to_log = "data/912-Cuts0-glass4-0.log"
+        self._header_parser = HeaderLogParser()
+
+    def test_get_flag(self):
+        correct_start_lines = [
+            "Gurobi 9.1.2 (linux64, gurobi_cl) logging started Fri Jul 30 13:53:48 2021",
+            "Compute Server job ID: 7a51ec41-f72d-4b31-95ba-28407986eda3",
+            "Gurobi Compute Server Worker version 9.5.0 build v9.1.1rc0 (linux64)",
+            "Logging started Fri Jul 30 13:53:48 2021",
+            "Logging started",
+            "Gurobi 10.0.0 (linux64, gurobi_cl) logging started Fri Jul 30 13:53:48 2021",
+        ]
+
+        wrong_start_lines = [
+            "Gurobi 900.1.2 (linux64, gurobi_cl) logging started Fri Jul 30 13:53:48 2021",
+            "Compute Server job: 7a51ec41-f72d-4b31-95ba-28407986eda3",
+            "Gurobi Compute Server Worker version 9.5.0 build v9.1.1rc0 linux64",
+            "Logging starte",
+        ]
+        for line in correct_start_lines:
```

Comment:
If we only see one of these lines in a log, the test should reflect that. I think it would be cleaner to have a parameterised test so that a fresh HeaderLogParser handles each test line.